test/jb_projects/challenge_handle.test.js

```diff
@@ -0,0 +1,92 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { getTimestamp } from '../helpers/utils';
+
+import { deployMockContract } from '@ethereum-waffle/mock-contract';
+
+import jbOperatoreStore from '../../artifacts/contracts/JBOperatorStore.sol/JBOperatorStore.json';
+
+describe('JBProjects::challengeHandle(...)', function () {
+  const PROJECT_HANDLE_1 = 'PROJECT_1';
+  const METADATA_CID = '';
+  const PROJECT_ID = 1;
+
+  async function setup() {
+    let [deployer, projectOwner, ...addrs] = await ethers.getSigners();
+
+    let mockJbOperatorStore = await deployMockContract(deployer, jbOperatoreStore.abi);
+
+    let jbProjectsFactory = await ethers.getContractFactory('JBProjects');
+    let jbProjectsStore = await jbProjectsFactory.deploy(mockJbOperatorStore.address);
+
+    return {
+      projectOwner,
+      deployer,
+      addrs,
+      jbProjectsStore,
+    };
+  }
+
+  it(`Should challenge handle successfully`, async function () {
+    const { projectOwner, deployer, addrs, jbProjectsStore } = await setup();
+
+    await jbProjectsStore
+      .connect(deployer)
+      .createFor(
+        /*owner=*/ projectOwner.address,
+        /*handle=*/ ethers.utils.formatBytes32String(PROJECT_HANDLE_1),
+        /*metadataCid=*/ '',
```

Comment:
This should be `METADATA_CID`, and since the variable name is obvious it should not need a param comment.

Reply:
similarly `handle` is not needed as a comment as well. It's clear from PROJECT_HANDLE_1 and that factor it's just formatting a string.